cmd/local/upgrade.go

```diff
@@ -20,15 +20,19 @@ func NewUpgradeCmd() *cobra.Command {
 				return err
 			}
 			ctx := cmd.Context()
-			bgLogger := local.SetupBackgroundLogger("upgrade")
+			close, err := local.Setup("upgrade")
```

Comment:
I feel like local.setup isnt a good name to understand what this does - does it setup the background logger? does it initialize the env?

Reply:
Looking at the setup func itself I think maybe something like SetupInfra can work here